Obedient-Lobsters/BangazonAPI
# EmployeeController GET, POST, PUT are all working.

# Description
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 Fixes #6 
 ## Type of change
 # Testing Instructions
In the directory BangazonAPI use the command ```dotnet run``` to start up the program. The server should start and run without errors. Once the program is running open up Postman and run the following commands: 

- select GET then paste ```localhost:5000/employee``` into the field and click send. The result should be an array of all the employees in the database.

- select GET then paste ```localhost:5000/employee/1``` or any other number that showed up in the previous query as EmployeeId and click send. The result should be only that object of the specified employee

- select POST then paste ```localhost:5000/employee``` into the field then click Body underneath the field then select raw and paste this snippet or make one similar ``` {
	"FirstName": "Fake",
	"LastName": "Person",
	"Email": "fakeemail@blah.com",
	"Supervisor": false,
	"DepartmentId": 1
}``` then click send. The result should be the new employee you made.

- select PUT then paste ```localhost:5000/employee/3``` or any other EmployeeId then do the same as the POST example and change the values then click send. You should get nothing back from this. When you run the GET query the employee you specified in your PUT query should show the updated information you gave it.

 # Checklist:
